libs/store/src/lib/infrastructure/persistence/store/entity-store-server/entity-store-server.ts

```diff
@@ -0,0 +1,264 @@
+import { HttpClient, HttpErrorResponse, HttpParams } from '@angular/common/http';
+import { Injectable, Optional } from '@angular/core';
+import { Observable, of, throwError } from 'rxjs';
+import { timeout, delay, catchError, map } from 'rxjs/operators';
+import { Update } from '@ngrx/entity';
+import { DefaultDataServiceConfig, EntityCollectionDataService, QueryParams, DataServiceError } from '@ngrx/data';
+
+import { EntityPath, EntityOperation } from '../../../../domain/public_api';
+import { EntityMetaOptionsService, EntityResourceMetaOptions, EntityMetaOptions } from '../../entity-options.service';
+
+import { HttpUrlGenerator } from './http-url-generator';
+
+export declare type HttpMethods = 'DELETE' | 'GET' | 'POST' | 'PUT' | 'PATCH';
+
+export interface RequestData {
+    method: HttpMethods;
+    url: string;
+    data?: any;
+    options?: any;
+}
+
+/**
```

Comment:
We dont need additional level `store/entity-store-server/` I would probably prefer instead of having this folder I would just use /store

Reply:
if you feel you have many files, you can move generators and builder to /store/<builder | rest | memory>

Reply:
removed `entity-store-server` folder